src/java/org/apache/cassandra/cql3/statements/schema/DropFunctionStatement.java

```diff
@@ -120,9 +121,28 @@ public Keyspaces apply(Keyspaces schema)
         if (!dependentAggregates.isEmpty())
             throw ire("Function '%s' is still referenced by aggregates %s", name, dependentAggregates);
 
+        String dependentMasks = keyspace.tables.stream()
+                                               .filter(table -> hasDependingMask(table, function))
+                                               .map(table -> table.name)
+                                               .collect(joining(", "));
+
+        if (!dependentMasks.isEmpty())
+            throw ire("Function '%s' is still referenced by column masks in tables %s", name, dependentMasks);
```

Comment:
I would add the keyspace name here "ks.table". Countless times I've amended error messages just bc a user had the same exact ks but: ks_dev, ks_testWhatever, ks_preProd, etc

Reply:
The `name` var is the qualified name of the function, so it already contains the name of the keyspace. That `name` var is created a few lines above, at the beginning of the method.

Reply:
Wfm Thx